### PR TITLE
Use css class to customize sticky ad top padding bar

### LINKED ISSUE #9927: Use css class to customize sticky ad top padding bar
**Reserved For write/speak/code event**

### Description
`<amp-sticky-ad>` has a top padding bar `amp-sticky-ad-top-padding` to make it looks better. However the element has a name start with `amp-*`, the name should be reserved for AMPElemnt only. 

We can't rename the element's name now because it can introduce a breaking change to existence pages. We can however provide an alternative way to customize the top padding bar by adding a css class to the element. 

Ref issue: #9283 

### The Fix

- Go to [/1.0/amp-sticky-ad.js](https://github.com/ampproject/amphtml/blob/master/extensions/amp-sticky-ad/1.0/amp-sticky-ad.js) file
- Find the `<amp-sticky-ad-top-padding>` element
- Add a css class to the element to enable customizing. 
- Test manually 
- Fix unit test. Make sure tests pass.
- Submit for review!

Once approved, your changes will be merged.  **⚡⚡⚡Congrats on making your first contribution to the AMP Project!⚡⚡⚡**  You'll be able to see it [live across the web soon](https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md)!

Thanks, and we hope to see more contributions from you soon.

### Questions?

If you have questions ask in this issue or on your Pull Request see the [How to get help](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#how-to-get-help) section of the Getting Started guide.

### PR DESCRIPTION
**Pull request by kellyk**
Fix #9927

### CODE REVIEW

File: extensions/amp-sticky-ad/1.0/amp-sticky-ad.js [hunk 1]
```diff
@@ -77,6 +77,7 @@ class AmpStickyAd extends AMP.BaseElement {
 
     const paddingBar = this.win.document.createElement(
         'amp-sticky-ad-top-padding');
+    paddingBar.classList.add('sticky-ad-top-padding');
```
Comment (zhouyx) [inline extensions/amp-sticky-ad/1.0/amp-sticky-ad.js]: class name should start with `amp-`, use 'amp-sticky-ad-top-padding' instead.

**Comment by jridgewell** [comment]
Let's update the documentation with this?

**Comment by zhouyx** [comment]
@jridgewell I can create another issue on updating doc.
@kellyk LGTM. merged.

**Comment by kellyk** [comment]
Sorry I didn't update the docs in time!

**Comment by zhouyx** [comment]
@kellyk No worry. Updating doc can be another first issue

**Pull request merged**